server.js

```diff
@@ -1202,9 +1202,10 @@ app.get('/get-task/:version(\\d+)', asyncMiddleware( async (req, res, next) => {
 
         // Not the most robust, but if we have completed and send out more than X requests for this match, stop requesting it.
         //
-        pending_matches[pending_matches.length - 1].game_count++;
+        match.game_count++;
```

Comment:
If we remove this, then we can keep an unfinished match alive until it really does finish. So the next match won't start too soon if this one is unknown status.

Reply:
Oh, that's a nice bug to fix 👍